### PR TITLE
Fixes for `convolve_fft` examples.

### PR DESCRIPTION
**Pull request by WilliamJamieson**
### Description

Several of the documentation examples in the `convolve_fft` documentation provide incorrect outputs. These have been corrected.

### CODE REVIEW

**Review by jdavies-st — COMMENTED** [review]
Since this presumably makes the doctests now pass, can you turn them on?  I.e. modify or remove

https://github.com/astropy/astropy/blob/45dbd1f3f8a2a119c41ca8c2912f77707b23bd4a/astropy/convolution/convolve.py#L51

File: astropy/convolution/convolve.py [hunk 1]
```diff
@@ -535,25 +531,24 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
     array([ 1.,  2.,  3.])
```
Comment (nden) [inline astropy/convolution/convolve.py:531]:
```suggestion
    array([1.,  2.,  3.])
```

Comment (nden) [reply astropy/convolution/convolve.py:531]: There's an extra space here and elsewhere in the examples

Comment (WilliamJamieson) [reply astropy/convolution/convolve.py]: Fixed in 8632901

**Review by nden — APPROVED** [review]

**Comment by nden** [comment]
tagging @larrybradley & @adamginsburg in case you want to look at the examples before this is merged

**Review by larrybradley — APPROVED** [review]

**Pull request merged**